### PR TITLE
SimpleSlider decimalScale bugfix and thousandSeparator inclusion

### PR DESCRIPTION
**Pull request by panaitsilviu**
When using SimpleSlider CustomTextField should be numeric.
image

### CODE REVIEW

File: components/SimpleSlider/SimpleSlider.js [hunk 1]
```diff
@@ -79,8 +80,9 @@ SimpleSlider.propTypes = {
     helperText: PropTypes.string,
     required: PropTypes.bool,
     decimalScale: PropTypes.number,
+    thousandSeparator: PropTypes.bool,
```
Comment (DCosti) [inline components/SimpleSlider/SimpleSlider.js:83]: This property is sent to CustomTextField component. In that component, this property is of type bool or string. Modify here also to be of type string or bool

Comment (DCosti) [reply components/SimpleSlider/SimpleSlider.js:83]: Add this property to d.ts also

Comment (panaitsilviu) [reply components/SimpleSlider/SimpleSlider.js]: I have done the suggested modifications.

File: components/SimpleSlider/SimpleSlider.d.ts [hunk 2]
```diff
@@ -67,6 +67,11 @@ export interface Props {
      * number of decimals to be displayed
      */
     decimalScale?: number;
+  
+    /**
+    * Add thousand separators on number; single character string or boolean true
```
Comment (DCosti) [inline components/SimpleSlider/SimpleSlider.d.ts:72]: You can add also: If true, the default is ","

**Review by DCosti — APPROVED** [review]

**Pull request merged**